CHANGELOG.md

```diff
@@ -49,7 +50,6 @@
 - `transformprocessor`: Add functions for conversion of scalar metric types (`gauge_to_sum` and `sum_to_gauge`) (#10255)
 - `dynatraceexporter`: Use min and max when provided in a data point for histograms (#10815)
 - `dynatraceexporter`: Truncate unmarshalable responses to avoid long log lines (#10568)
-- `scrapertest`: Add `IgnoreResourceAttributeValue` option to metric comparison (#10828)
```

Comment:
Is this removed unintentionally?

Reply:
Argh.. unintentional! I guess happened as part of resolving a merge conflict.